app/views/agent/partials/RegistrationPartial.scala.html

```diff
@@ -51,41 +51,39 @@
 }
 
 @cancelRegistration = {
-    <div id="cancel-vat" class="column-one-third">
-        <h3 class="heading-small">
-            @cancelVatLink
-        </h3>
-        <p>@messages(cancelVatContentKey)</p>
+    <div class="govuk-grid-column-one-third" id="cancel-vat">
+        <h3 class="govuk-heading-s">@cancelVatLink</h3>
```

Comment:
Can you add class `govuk-!-margin-bottom-1` to all `h3`s in this partial. This is so the text below it sits neatly under the heading. We had to do the same thing on vat-summary so that each little section fit together a bit better.